src/tests/functional/plugin/shared/src/behavior/plugin/configuration_tests.cpp

```diff
@@ -214,4 +213,25 @@ TEST_P(SetPropLoadNetWorkGetPropTests, SetPropLoadNetWorkGetProperty) {
     }
 }
 
+TEST_P(AutoMultiSetPropLoadNetWorkGetPropTests, AutoMultisetConfigAndLoadNetWorkSetGetPropertyTest) {
```

Comment:
why do we need AUTO tests in shared common tests?
They are by definition should be located in AUTO plugin folder.

Shared folder are allowed to contain only generic tests which are valid for all plugins